pgjdbc/src/main/java/org/postgresql/jdbc/SSLNegotiation.java

```diff
@@ -0,0 +1,25 @@
+package org.postgresql.jdbc;
+
+public enum SSLNegotiation {
```

Comment:
WDYT of `SslNegotiation`?

See https://google.github.io/styleguide/javaguide.html#s5.3-camel-case